corgi/collectors/prod_defs.py

```diff
@@ -9,7 +12,7 @@ class ProdDefs:
     GSSAPI_AUTH = HTTPSPNEGOAuth()
 
     @classmethod
-    def get_product_definitions(cls) -> dict:
+    def get_enterprise_product_definitions(cls) -> dict:
```

Comment:
nitpick: not a fan of 'enterprise' denotation ... alternately (if community is a mode) then we could route between func impl based on mode instead of incurring different codepaths - note I make this suggestion not really understanding potential for violence in the codebase.